### PR TITLE
#161966602 Add coveralls badge

### PR DESCRIPTION
**Pull request by salma-nyagaka**
#### What does this PR do?
Add coveralls to the project and the badge to README.md
Update the README.md file to include how to run the application
#### Description of Task to be completed?
Integrate Coveralls code coverage service
Guidelines for running the application
#### How should this be manually tested?
- Go to the README.md and confirm if the badge is there
#### Any background context you want to provide?
- This is to ensure we are properly informed just how well our code base is tested which in turn provides quality assurance for the product and provide instructions on how to set up and run the application
#### What are the relevant pivotal tracker stories?
[#161966602](https://www.pivotaltracker.com/n/projects/2225606)
[#162236747](https://www.pivotaltracker.com/n/projects/2225606)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] At least 2 people have reviewed my PR
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by verenceLola** [comment]
Next time please tag reviewers.

**Comment by SnyderMbishai** [comment]
Good work on this!

**Review by SnyderMbishai — DISMISSED** [review]

**Review by verenceLola — DISMISSED** [review]
Good work @salma-nyagaka for delivering your work in time.

**Review by Mnickii — DISMISSED** [review]
LGTM. Rebase your PR

**Review by SilasKenneth — DISMISSED** [review]

**Review by SilasKenneth — DISMISSED** [review]

**Review by SilasKenneth — DISMISSED** [review]

**Comment by SnyderMbishai** [comment]
Good work, please check the PR, there should be a question mark after 'How should this be manually tested'.

**Comment by kipropbrian** [comment]
This commit has issues too, as do the others. It appears you did not rebase correctly

@SilasKenneth  Did you checkout to develop first before checking out to this branch and making the changes

File: authors/apps/authentication/backends.py [hunk 1]
```diff
@@ -1,10 +1,62 @@
...
+
+from .models import User, BlacklistedToken
+
+
+class JWTAuthentication(authentication.BaseAuthentication):
+    """
+    This class gets the authorization header gotten
+    from the request header with the keyword Token in
+    it
+    """
```
Comment (Mnickii) [inline authors/apps/authentication/backends.py]: These changes are out of scope of your PT story. Make sure this PR only addresses the story assigned

**Review by Mnickii — CHANGES_REQUESTED** [review]
Your PR has too many changes unrelated to adding a coveralls badge and updating the readme. Fix this

**Review by actlikewill — COMMENTED** [review]
Hope you were able to solve the conflicts ...

**Review by Mnickii — APPROVED** [review]
LGTM

**Pull request merged**